packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -94,6 +94,7 @@ export class ProxyController extends Controller<ProxyControllerEventMap> {
 							unsafePreventEviction: true,
 						},
 					},
+					stripCfConnectingIp: false,
```

Comment:
It is worth adding a comment here to explain why this is necessary?
And in miniflare.ts too?

Reply:
[fb431dc](https://github.com/cloudflare/workers-sdk/pull/7914/commits/fb431dc1d811ef1aebe1d02348b9461663f552da)